src/Microsoft.DotNet.Build.Tasks/PackageFiles/Roslyn.Common.props

```diff
@@ -4,8 +4,11 @@
     <RoslynVersion>2.0.0-rc</RoslynVersion>
     <RoslynPackageName>Microsoft.Net.Compilers</RoslynPackageName>
     <RoslynTargetsPath>$(ToolRuntimePath)</RoslynTargetsPath>
-    <CSharpCoreTargetsPath Condition="'$(CSharpCoreTargetsPath)' == ''">$(RoslynTargetsPath)$(RoslynPackageName)\$(RoslynVersion)\tools\Microsoft.CSharp.Core.targets</CSharpCoreTargetsPath>
-    <VisualBasicCoreTargetsPath Condition="'$(VisualBasicCoreTargetsPath)' == ''">$(RoslynTargetsPath)$(RoslynPackageName)\$(RoslynVersion)\tools\Microsoft.VisualBasic.Core.targets</VisualBasicCoreTargetsPath>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(OS)' == 'Windows_NT'">
```

Comment:
Thanks @jaredpar. While that does suck I can certainly understand it. On a related note do you guys currently produce two separate packages, one for desktop and on for NETCore? Or do you guys have a single package for both yet?